public/locales/en/translation.json

```diff
@@ -431,7 +431,8 @@
           "label": "Share",
           "tooltip": "Modify access",
           "editModeTooltip": "Please save or discard current modifications before changing the scenario access permissions",
-          "noReadSecurityPermissionTooltip": "You are not allowed to share or see the security details of this resource"
+          "noReadSecurityPermissionTooltip": "You are not allowed to share or see the security details of this resource",
+          "noAccessToBaseDatasetsTooltip": "Cannot share this scenario because you cannot share its base dataset \"{{restrictedDatasetId}}\". Please request the \"admin\" role on this dataset first, or ask the dataset owner to share this scenario for you."
```

Comment:
If it's possible, can we display the dataset name instead of its id? (at least when the connected user has access to it)
Same thing for the label `disabledUserTooltip` below